### PR TITLE
Create a Log Integration Update

### PR DESCRIPTION
**Pull request by alai97**
### What does this PR do? What is the motivation?

Adds a screenshot of the Export Pipeline icon + updates instructions in the Create a Log Integration doc page.

Chat with Emily on Slack.

### Merge instructions

- [x] Please merge after reviewing

### CODE REVIEW

**Review by emarsha94 — APPROVED** [review]

**Review by hestonhoffman — APPROVED** [review]

File: content/en/developers/integrations/log_integration.md [hunk 1]
```diff
@@ -119,13 +119,26 @@ To easily navigate the facet list, facets are grouped together. For fields speci
...
-Be sure to include sample raw logs with all the attributes you expect to be sent into Datadog by your integration. Raw logs comprise the raw messages generated directly from the source before they have been ingested by Datadog.
+{{< img src="developers/integrations/export_pipeline.png" alt="Click the Export Pipeline icon to export your log pipeline in Datadog" width="50%">}}
 
-Once reviews are complete, Datadog creates and deploys the new log integration assets.
+Include sample raw logs with all the attributes you expect to be sent into Datadog by your integration. Raw logs comprise of the raw messages generated directly from the source before they have been ingested by Datadog.
+
+Exporting your log pipeline includes two YAML files:
+
+- One with the log pipeline, which includes custom facets, attribute remappers, and grok parsers.
+- One with the raw example logs with an empty result. Depending on your browser, you may need to adjust your settings to allow file downloads.
```
Comment (eho1307) [inline content/en/developers/integrations/log_integration.md:133]:
```suggestion
- One with the raw example logs with an empty result. 

Note: Depending on your browser, you may need to adjust your settings to allow file downloads.
```

File: content/en/developers/integrations/log_integration.md [hunk 2]
```diff
@@ -119,13 +119,26 @@ To easily navigate the facet list, facets are grouped together. For fields speci
...
+
+Once you've downloaded these files, navigate to your [integration's pull request][22] on GitHub and add them in the **Assets** > **Logs** directory. If a Logs folder does not exist yet, you can create one.
+
+Validations are run automatically in your pull request. A common log pipeline validation error is not populating the `id` field in both YAML files. Ensure that the `id` field matches the `app_id` field in your integration's `manifest.json` file to connect your pipeline to your integration.
```
Comment (eho1307) [inline content/en/developers/integrations/log_integration.md:137]:
```suggestion
Validations are run automatically in your pull request. 

Two common validation errors are:
1. The `id` field in both YAML files - Ensure that the `id` field matches the `app_id` field in your integration's `manifest.json` file to connect your pipeline to your integration. 
2. Not providing the result of running the raw logs you provided against your pipeline. If the resulting output from the validation is accurate, take that output and add it to the `result` field in the YAML file containing the raw example logs.
```

File: content/en/developers/integrations/log_integration.md [hunk 3]
```diff
@@ -119,13 +119,26 @@ To easily navigate the facet list, facets are grouped together. For fields speci
...
+Exporting your log pipeline includes two YAML files:
+
+- One with the log pipeline, which includes custom facets, attribute remappers, and grok parsers.
+- One with the raw example logs with an empty result. Depending on your browser, you may need to adjust your settings to allow file downloads.
+
+Once you've downloaded these files, navigate to your [integration's pull request][22] on GitHub and add them in the **Assets** > **Logs** directory. If a Logs folder does not exist yet, you can create one.
+
+Validations are run automatically in your pull request. A common log pipeline validation error is not populating the `id` field in both YAML files. Ensure that the `id` field matches the `app_id` field in your integration's `manifest.json` file to connect your pipeline to your integration. 
+
+Another common validation error is not providing the result of running the raw logs you provided against your pipeline. If the resulting output from the validation is accurate, take that output and add it to the `result` field in the YAML file containing the raw example logs.
```
Comment (eho1307) [inline content/en/developers/integrations/log_integration.md:139]:
```suggestion
```

**Review by eho1307 — COMMENTED** [review]
Suggestions

**Review by eho1307 — APPROVED** [review]

**Pull request merged**